.github/workflows/validate-owasp-metadata.yaml

```diff
@@ -0,0 +1,24 @@
+name: Validate OWASP entity metadata
+
+on:
+  pull_request:
+    paths:
+      - '*.owasp.yaml'
+  push:
+    paths:
+      - '*.owasp.yaml'
+
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.repository }}-${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  validate-metadata:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+
+      - name: Validate metadata file
+        uses: owasp/nest-schema/.github/actions/validate@a733198b4a942eb12d3ee8629cd9e0d409b1b2b9
```

Comment:
>Why would there be dependabot settings for a wiki page?

@davewichers this line adds validation workflow that needs to be kept up to date, hence the dependabot config (see https://github.com/OWASP/www-project-vulnerable-web-applications-directory/pull/152#pullrequestreview-3265724315 for more details)